spec/puppet-server-lib/puppet/jvm/config_spec.rb

```diff
@@ -4,16 +4,46 @@
 
 describe Puppet::Server::Config do
   context "SSL context" do
-    it "is configured with :localcacert and not :cacert" do
+    before :each do
       Puppet[:hostcert] = "spec/fixtures/localhost-cert.pem"
       Puppet[:hostprivkey] = "spec/fixtures/localhost-privkey.pem"
-      Puppet[:cacert] = "SHOULD/NOT/BE/USED"
       Puppet[:localcacert] = "spec/fixtures/ca-cert.pem"
+    end
+
+    it "is configured with :localcacert and not :cacert" do
+      Puppet[:cacert] = "SHOULD/NOT/BE/USED"
+
       begin
-        Puppet::Server::Config.ssl_context
+        ssl_context = Puppet::Server::Config.ssl_context
```

Comment:
I might not follow, I don't think I define it more than once per test example/it block

Reply:
This particular test doesn't use the `ssl_context` variable, so this line could go back to how it was before, since subsequent tests set the `ssl_context` variable themselves.

Reply:
You're right, this variable is unused and is an artifact of how I started adding to the existing test and then split out the second test. I can remove it.

But I do want to note that this is local variable and is not visible to any of the other `it` blocks.